dockerfiles/che/Dockerfile

```diff
@@ -40,4 +40,4 @@ ENTRYPOINT ["/entrypoint.sh"]
 RUN mkdir /logs /data && \
```

Comment:
I don't remember why this instruction couldn't be merged with previous RUN instruction ?

Reply:
@benoitf I don't think we have ever discussed that

Reply:
AFAIK It was discussed there https://github.com/eclipse/che/pull/6200#pullrequestreview-61541153

Reply:
Anyway I can't see any problem to do it. But it should be carefully tested.